src/compiler/utilities.ts

```diff
@@ -8814,7 +8814,108 @@ function createComputedCompilerOptions<T extends Record<string, CompilerOptionKe
 }
 
 /** @internal */
-export const computedOptions = createComputedCompilerOptions({
+export const computedOptions: {

```

Comment:
This is really really unfortunate. I'm not sure what to do about this one.

Reply:
Oops, I missed it in my pass through of const objects :,). Very unfortunate indeed

Reply:
I still hope there's something less complicated we can do here; I was hoping for a bunch of overloads on a single function instead, but `convertToTSConfig` iterates over this...

Reply:
@andrewbranch do you have suggestions?

Reply:
Can this be value on "CommandLineOption" in commandLineParser so its accessible via that?

Reply:
https://github.com/microsoft/TypeScript/commit/17efd55bbc1d2e12f29f2807d2c57a5112bbae5e